.gitignore

```diff
@@ -137,3 +137,11 @@ dmypy.json
 
 # MacOs
 **/.DS_Store
+
+
+tests/resources/procedures/updated_class_models/
+
+tests/resources/procedures/U19_examples/
+
+tests/resources/procedures/updated_class_models/
+
```

Comment:
Why ignore these?

Reply:
These are my outputs, didnt think it was necessary to upload them.